Onion/Helpers/ConfigurationManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Onion.Helpers
+{
+    public class ConfigurationManager
+    {
+        public static string LOG4NET_PATH { get; set; }
```

Comment:
Azure has storage capabilities for application paths like these, we should research and be prepared to use them...